🌐 Add Portuguese translation for `docs/pt/docs/tutorial/dependencies/index.md`

Joao-Pedro-P-Holanda opened the pull request:
Adding translation for `docs/pt/docs/tutorial/dependencies/index.md` and copying images in `docs/en/docs/img/tutorial/dependencies` into `docs/pt/docs/img/tutorial/dependencies`

Joao-Pedro-P-Holanda committed:
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/dependencies/index.md`

Copying images in docs/en/docs/img/tutorial/dependencies into docs/pt/docs/img/tutorial/dependencies

tiangolo commented:
📝 Docs preview for commit 1b89d4d9bce8ff6297d672407cc027cf4c49e1aa at: https://d5a57b6e.fastapitiangolo.pages.dev

ceb10n commented on docs/pt/docs/img/tutorial/dependencies/image01.png:
Why did you added this img?

Other translations like `ko` and `ja` didn't copy this file to their lang folder

Joao-Pedro-P-Holanda replied:
Reverted the addition, i added only because on local preview it didn't showed up, but turned out i was not building all of the documentation.

ceb10n commented on docs/pt/docs/img/tutorial/dependencies/image02.png:
Why did you added this img?

Other translations like `ko` and `ja` didn't copy this file to their lang folder

ceb10n commented on docs/pt/docs/tutorial/dependencies/index.md:
```diff
@@ -0,0 +1,353 @@
+
+E então retorna um `dict` contendo esses valores.
+
+!!! info "Info"
```
Most of the pt translations of !!! info is translated to "Informação"

Joao-Pedro-P-Holanda committed:
chore: addressing 1st review changes

changing "Info" for "Informação"

removing duplicated images, keeping the same path on line 268

tiangolo commented:
📝 Docs preview for commit 0d776017f73aed365381c5e99b86b7c20acc33e3 at: https://58465a64.fastapitiangolo.pages.dev

alejsdev commented:
Cool! Thank you @Joao-Pedro-P-Holanda @ceb10n :smile_cat: :rocket:
